data/test_validate/stories.md

```diff
@@ -0,0 +1,16 @@
+## Greting path
+* greet
+    - utter_greet
+    - utter_question
```

Comment:
Same again, use the `default_stories_file` (although you'll have to run the `StoryFileReader` on it too)